### PR TITLE
Create basic views

### PR DESCRIPTION
**Pull request by raorao**

### CODE REVIEW

File: app/controllers/index.rb [hunk 1]
```diff
@@ -6,9 +6,16 @@
 enable :sessions
 
 get '/' do
+  @all_users = User.all
   erb :index
 end
 
+get '/users/:id' do
+  @local_user = User.find(params[:id])
+  erb :users
```
Comment (nscricco) [inline app/controllers/index.rb:15]: maybe it makes more sense for this to be singular?

**Comment by nscricco** [comment]
looks good, fix that one tense issue (that I am prob just me being picky)

File: app/controllers/index.rb [hunk 2]
```diff
@@ -6,9 +6,16 @@
 enable :sessions
 
 get '/' do
+  @all_users = User.all
   erb :index
 end
 
+get '/users/:id' do
+  @queried_user = User.find(params[:id])
```
Comment (zspencer) [inline app/controllers/index.rb:14]: Why not just call this `@user`?

File: app/views/index.erb [hunk 3]
```diff
@@ -4,3 +4,8 @@
   Welcome, <%= current_user.firstname %>!
   <a href='/logout'>Logout</a>
 <% end %>
+
+<div id="title">all users</div>
+<% @all_users.each do |one_user| %>
+  <div><a href="/users/<%= one_user.id %>"><%= one_user.username %></a></div>
+<% end %>
```
Comment (zspencer) [inline app/views/index.erb:11]: If you change what `all_users` contains, perhaps by paginating it, would you need to change the variable name?

**Pull request merged**